_posts/2018-11-16-webhooks-about.markdown

```diff
@@ -0,0 +1,47 @@
+---
+permalink: v1/webhooks/about
+title: Webhooks
+layout: sidebar
+---
+
+<h1 class="block">
+    Webhooks
+</h1>
+
+## About
+Webhooks are a way to get notified when events happen in Gusto so that you do not need to come up with a polling strategy for refreshing data in your app.
+
+Webhooks trigger on certain events and send an HTTP POST payload to the URL that is registered by you for that event. You can limit which events trigger a webhook for your registered URLs.
+
+An **event** can be an employee being created for one of your companies, or the company itself being provisioned where access to their events were granted to you. You can create alerts in your app for running payroll based on the pay schedule, and update it if anything changes.
+
+A **payload** is the body of the request that will be sent to your app when an event happens. The top-level attributes describe the resources and provide timestamps to compare to other payloads. Every payload has an `entity_attributes` value that is the serialized entity which matches what you would have gotten if you had used our public API to get information on that entity. This way you do not need to query our API every time an event occurs.
+
+## Registering
```

Comment:
Add a section to explain the attributes of the top-level payload? I think `Resource` and `Entity` specifically might be confusing to developers:
```
{
  "event_type": "company.provisioned",
  "resource_type": "Company",
  "resource_id": 7757500908984137,
  "entity_type": "Company",
  "entity_id": 7757500908984137,
  "timestamp": 1533606328,
  "entity_attributes": {...}
}
```

Reply:
Added 👍